src/settings.c

```diff
@@ -121,6 +125,17 @@ settings_handle_read_all (XdpSettings           *object,
 
       g_variant_builder_add (&builder, "{s@a{sv}}", key, g_variant_dict_end (&dict));
     }
+
+  if (namespace_matches ("org.gnome.fontconfig", arg_namespace))
+    {
+      GVariantDict dict;
+
+      g_variant_dict_init (&dict, NULL);
+      g_variant_dict_insert_value (&dict, "timestamp", g_variant_new_int32 (self->fontconfig_timestamp));
```

Comment:
Probably best to keep timestamps in int64.

Reply:
can't do that, since xsettings only knows string, bool and int, and we're still kinda limited by those constraints on the gtk side

Reply:
The GTK MR for it already used int64: https://gitlab.gnome.org/GNOME/gtk/merge_requests/241

Even if it gets truncated by GTK in the short term might as well make the DBus API use the right type.

Reply:
I'd rather rename it. The thing is not a real timestamp in any way

Reply:
I've done that now